en_US/LC_MESSAGES/django.po

```diff
@@ -4048,437 +4181,404 @@ msgstr "There is no topic data available for the current selections."
-msgid "Close"
-msgstr "Close"
-
-#: kitsune/questions/jinja2/questions/question_details.html:348 kitsune/search/forms.py:183
-msgid "Tags"
-msgstr "Tags"
+#: kitsune/questions/jinja2/questions/question_details.html:523
+#, fuzzy
+msgid "Edit details"
+msgstr "Edit Description"
 
-#: kitsune/questions/jinja2/questions/question_details.html:388 kitsune/tags/forms.py:121
+#: kitsune/questions/jinja2/questions/question_details.html:533 kitsune/wiki/jinja2/wiki/document.html:120
 #, fuzzy
-msgid "Add"
-msgstr "Add-ons"
+msgid "Related Articles"
+msgstr "Release Notes"
 
-#: kitsune/questions/jinja2/questions/question_details.html:401
+#: kitsune/questions/jinja2/questions/question_list.html:9
 #, fuzzy
-msgid "Related"
-msgstr "Created"
+msgid "Multiple Products"
+msgstr "Products"
 
-#: kitsune/questions/jinja2/questions/question_details.html:421
-msgid "Invite to share screen"
-msgstr "Invite to share screen"
+#: kitsune/questions/jinja2/questions/question_list.html:60
+#, fuzzy, python-format
+msgid "%(recent_unanswered_count)s question in the last 24 hours has no reply. <a href=\"%(url)s\">Help solve it!</a>"
+msgid_plural "%(recent_unanswered_count)s questions in the last 24 hours> have no reply. <a href=\"%(url)s\">Help solve them!</a>"
+msgstr[0] "<strong>%(recent_replied_count)s tweets</strong> in the last"
+msgstr[1] "<strong>%(recent_replied_count)s tweets</strong> in the last"
 
-#: kitsune/questions/jinja2/questions/question_details.html:451
-#, python-format
-msgid ""
-"In order to prevent spam, we are limiting the number of answers that can be posted within a short period of time. If you want to post more please create a post on the <a href=\"%(url)s"
-"\">contributors forum</a> requesting that you be white-listed."
-msgstr ""
-"In order to prevent spam, we are limiting the number of answers that can be posted within a short period of time. If you want to post more please create a post on the <a href=\"%(url)s"
-"\">contributors forum</a> requesting that you be white-listed."
+#: kitsune/questions/jinja2/questions/question_list.html:68
+#, fuzzy, python-format
+msgid "%(recent_unanswered_count)s question in the last 24 hours has no reply. <a href=\"/get-involved\">Learn how to help solve it!</a>"
+msgid_plural "%(recent_unanswered_count)s questions in the last 24 hours> have no reply. <a href=\"/get-involved\">Learn how to help solve them!</a>"
+msgstr[0] "<strong>%(recent_replied_count)s tweets</strong> in the last"
+msgstr[1] "<strong>%(recent_replied_count)s tweets</strong> in the last"
 
-#: kitsune/questions/jinja2/questions/email/new_answer.html:22 kitsune/questions/jinja2/questions/question_details.html:462
+#: kitsune/questions/jinja2/questions/question_list.html:78
 #, fuzzy
-msgid "Post a Reply"
-msgstr "Post a reply"
+msgid "0 questions in the last 24 hours have no reply. Good job!"
+msgstr "Found <strong>{n}</strong> result for <strong>{q}</strong> in <strong>{l}</strong>"
```

Comment:
This one and the one followed (maybe more later) is very hard to tell what the variables are for. Is there a place to put comment somewhere? If not, how do we document this so I can send comms to localizers - this is a workaround but not ideal. Not everyone is on the mailing list, especially new contributors.

Reply:
Similar, this should be changed on the translation side.